portal-web/test/functional/com/liferay/portalweb/macros/json/util/JSONUtil2.macro

```diff
@@ -142,18 +142,25 @@ definition {
 
 	@summary = "returns ${userEmail}:${password} to be used in JSON calls ONLY"
 	macro formatJSONUser {
-		var defaultUserEmail = JSONUtil2.getDefaultUserEmail();
+		var defaultUserEmail = JSONUtil2.getDefaultUserEmail(specificURL = ${specificURL});
```

Comment:
We are using domain here not a specificURL, can you rename the variable to this so its less misleading